tests/test_erosita.py

```diff
@@ -725,176 +725,6 @@ def test_successful_download(self):
         self.mock_copyfileobj.assert_has_calls(expected_calls_to_copyfileobj)
         self.mock_gzip.open.assert_called_once_with(local_dir + to_down, 'rb')
         self.mock_remove.assert_called_once_with(local_dir + to_down)
-    
```

Comment:
Because eRASS1DE was updated to also download background maps, this would mean adding a lot to this test - which was already quite convoluted. I just decided to get rid of this test, which I hope is fine because the coverage is still 63% without it. Happy to update this test though if you would like.